src/Api/Service/MyParcelApiService.php

```diff
@@ -46,6 +44,16 @@ public function __construct(?array $config, ClientAdapterInterface $clientAdapte
         parent::__construct($clientAdapter);
     }
 
+    /**
+     * @return string
+     */
+    public static function getVersion(): string
+    {
+        $composerJson = json_decode(file_get_contents(__DIR__ . '../../../../composer.json'), false);
```

Comment:
Kunnen we deze op zijn minst nog ergens bewaren dat hij niet altijd dat bestand moet lezen?

Reply:
dit hoeft (/mag) niet static en hoort niet thuis in de ApiService